vendor.conf

```diff
@@ -34,7 +34,7 @@ github.com/json-iterator/go 1.1.5
 github.com/konsorten/go-windows-terminal-sequences v1.0.1
 github.com/matttproud/golang_protobuf_extensions v1.0.0
 github.com/Microsoft/go-winio v0.4.14
-github.com/Microsoft/hcsshim 079e6c8fa2c3b641844e7f217d58a5a57b3a6440
+github.com/Microsoft/hcsshim 8a8b9187a4b552fa3e853b97e7cc39aa3379f4ca https://github.com/katiewasnothere/hcsshim.git
```

Comment:
This will be updated when https://github.com/microsoft/hcsshim/pull/719 is merged.